### ISSUE TITLE
[Feat] CI 시크릿 설정 및 CD 설정을 위한 파일 분리

### ISSUE THREAD

**Issue by ohyuchan123** [DaengPlace/BackEnd#55]
## 어떤 기능인가요?

> 추가하려는 기능에 대해 간결하게 설명해주세요

CI/CD 파이프라인 구축을 위한 GitHub Actions 워크플로우 설정 및 파일 분리

## 작업 상세 내용

- [x] GitHub Secrets 설정
- [x] Slack Webhook URL 설정 분리
